Dockerfile.no_ML

```diff
@@ -30,7 +30,8 @@ RUN apt-get update && apt-get install -y \
 
 COPY memgraph-${TARGETARCH}.deb .
 
-RUN dpkg -i memgraph-${TARGETARCH}.deb && rm memgraph-${TARGETARCH}.deb
+RUN dpkg -i memgraph-${TARGETARCH}.deb && rm memgraph-${TARGETARCH}.deb && rm /usr/lib/memgraph/query_modules/schema.so  \
+&& rm /usr/lib/memgraph/query_modules/example_cpp.so
```

Comment:
This shouldn't be needed in the next release but currently (since mage tests don't run on memgraph/master but 2.13 version= two regex tests crash without this.

Reply:
Aha, okay!